prelude/runtime/posix.c

```diff
@@ -297,17 +290,14 @@ int64_t SKIP_posix_isatty(int64_t fd) {
   return (char)rv;
 }
 
-int64_t SKIP_posix_mkstemp(char* template_obj) {
-  char* template = sk2c_string(template_obj);
```

Comment:
It turns out that this function relies on `sk2c_string` to make a copy since `mkstemp` will overwrite its argument. This broke sktest in a way that made it not run any tests. Fixed in latest update.